### ISSUE TITLE
db_directory_structure - folders rights assignment

### ISSUE THREAD

**Issue by cborisa** [biemond/biemond-oradb#212]
Currently the function db_directory_structure uses FileUtils.chmod and FileUtils.chown to set access rights to Oracle folders (oracle_base), (ora_inventory). However, it doesn't set up access rights to subfolders, causing installation of an Oracle client to crash.
chmod and chown should be replaced with chmod_R and chown_R

**Comment by biemond** [comment]
No, I don't think this is the real issue here, especially with oracle_base  & ora inventory you need to be careful , it should also work with grid etc.

**Comment by cborisa** [comment]
I'm having this issue in RHEL 7.4 when using oradb::client. 
I see that the folders for oracle_base and oracle_inventory are created. Oracle base gets access rights 0755 for a defines user and group_install. However all subfolders of Oracle_base remain with Root(Puppet runs with root). And this cause client installer to return an error as it's not able to create oracle_home (I have it inside oracle_base).
As soon as I change the function as suggested, the folders get correct access rights and the installation works

**Comment by biemond** [comment]
I think you are doing something wrong because I don't create any directory under oracle base, it is empty for me and the oracle installer will create them
So your oracle base can be an existing one & should match group role like oinstall at least or you provided the wrong params
